manifests/c/c0re100/qBittorrent-Enhanced-Edition/release-4.3.9.10/c0re100.qBittorrent-Enhanced-Edition.yaml

```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: c0re100.qBittorrent-Enhanced-Edition
+PackageVersion: release-4.3.9.10
```

Comment:
```suggestion
PackageVersion: 4.3.9.10
```

Reply:
@ItzLevvie I think these conversations will need to be marked as resolved before it can be merged